src/reducers/list-reducer.ts

```diff
@@ -10,8 +10,10 @@ import {
   RESOURCE_LIST,
   RESOURCE_LIST_SUCCESS,
   RESOURCE_LIST_FAILURE,
+  RESOURCE_DEL_SUCCESS,
 } from '../action-types';
 import { INITIAL_RESOURCE } from './initial-resource';
+import { SSL_OP_MICROSOFT_BIG_SSLV3_BUFFER } from 'constants';
```

Comment:
`SSL_OP_MICROSOFT_BIG_SSLV3_BUFFER`? 🤔

Reply:
This is VSCode adding those I thought I removed all of them.

Reply:
Removed